ol-keycloak/oltheme/src/main/resources/theme/ol/login/messages/messages_en.properties

```diff
@@ -1,4 +1,4 @@
-doRegisterSubmit=Next
+doRegisterSubmit=Sign Up
```

Comment:
Can you reuse the OOTB message `doRegister`?

Reply:
I'd like to leave it because who knows if this will change again.